samples/v1/types_config.go

```diff
@@ -64,6 +64,17 @@ type ConfigSpec struct {
 	// content but the operator will not recreate(or update) anything
 	// listed here.
 	SkippedTemplates []string `json:"skippedTemplates,omitempty" protobuf:"bytes,6,opt,name=skippedTemplates"`
+
+	// helmchart list specifies the list of helmcharts that the samples operator will install
+	HelmChartList []HelmChart `json:"helmChartList,omitempty" protobuf:"bytes,7,opt,name=helmChartList"`
+}
+
+type HelmChart struct {
+	URL string `json:"url,omitempty"`
+
+	Namespace string `json:"namespace,omitempty"`
+
+	ReleaseName string `json:"releaseName,omitempty"`
```

Comment:
godoc these fields please.  Also is there any validation we can put in place?

should these be required fields?